examples/code/async/search_with_error_handling/run_search_with_error_handling.sh

```diff
@@ -1,2 +1,16 @@
-jbuilder build search_with_error_handling.exe
-./_build/default/search_with_error_handling.exe -servers localhost,api.duckduckgo.com "Concurrent Programming" OCaml
+  $ jbuilder build search_with_error_handling.exe
+  $ ./_build/default/search_with_error_handling.exe -servers localhost,api.duckduckgo.com "Concurrent Programming" OCaml
+  Concurrent Programming
+  ----------------------
+  
+  DuckDuckGo query failed: Unexpected failure
```

Comment:
Can we do something to mark these broken examples so we can go back to them and fix them later, maybe?